windows/src/engine/keyman32/kmhook_getmessage.cpp

```diff
@@ -252,13 +252,9 @@ LRESULT _kmnGetMessageProc(int nCode, WPARAM wParam, LPARAM lParam)
 	 Handle WM_UNICHAR messages for RichEdit control -- should we test RichEdit version?
 	*/
 
-	if(mp->message == WM_UNICHAR && Addin_ShouldProcessUnichar(mp->hwnd))
+	if(mp->message == WM_UNICHAR)
```

Comment:
@mcdurdin I blindly removed addins - I don't know what they did. Should `WM_UNICHAR` be removed totally as it seems addins was the only function interested in the event. ?

Reply:
I think `WM_UNICHAR` is very much dead at this point. But perhaps it should be removed entirely in a separate PR (and also remove Keyman_ForceKeyboard in a separate PR too) -- that way we have PR# to reference them for potential future questions.

Reply:
PR for removing `WM_UNICHAR` https://github.com/keymanapp/keyman/issues/8677